packages/babel-plugin/src/css-prop/__tests__/object-literal.test.tsx

```diff
@@ -492,21 +500,8 @@ describe('css prop object literal', () => {
         <div css={{ color: 'blue', ...mixin() }}>hello world</div>
       `);
 
-    expect(actual).toInclude('.cc-hash-test{color:blue;color:red}');
-  });
-
-  xit('should transform object with argument arrow function variable', () => {
```

Comment:
Is this test duplicated somewhere? That's why we removed it right? Or we don't need it right now?

Reply:
it's not - i figured we can just remove these for now and then re-write them :) they've never been implemented so we might as well just start fresh! what do you think?

Reply:
Yups works for me